.github/workflows/build.yml

```diff
@@ -98,6 +98,8 @@ jobs:
           BASE_IMAGE_TAG=${{ matrix.baseImageTag }}
           SBT_VERSION=${{ steps.get_sbt_version.outputs.VERSION }}
           SCALA_VERSION=${{ matrix.scalaVersion }}
+        platforms: ${{ matrix.platforms }}
```

Comment:
Since we only test on the current platform we don't need this
In the example on the docker website there is a comment that explains that the current platform build we be cached and reused in the push step.

Reply:
Looks like this is also not supported (see build result)